components/navigable-menu/index.js

```diff
@@ -62,7 +62,7 @@ class NavigableMenu extends Component {
 		const { children, ...props } = this.props;
 
 		return (
-			<div ref={ this.bindContainer } { ...omit( props, [ 'orientation', 'onNavigate' ] ) } onKeyDown={ this.onKeyDown }>
+			<div ref={ this.bindContainer } { ...omit( props, [ 'orientation', 'onNavigate', 'deep' ] ) } onKeyDown={ this.onKeyDown }>
```

Comment:
That's a long line.

Also warning on `jsx-a11y/no-static-element-interactions`